docs/data_driven_testing.adoc

```diff
@@ -130,15 +130,15 @@ Note that unrolling has no effect on how the method gets executed; it is only an
 Depending on the execution environment, the output will look something like:
 
 ----
-maximum of two numbers[0]   PASSED
-maximum of two numbers[1]   FAILED
+maximum of two numbers [a: 1, b: 3, c: 3, iterationIndex: 0]   PASSED
```

Comment:
I think we can just use `#` instead of `iterationIndex`
`[a: 1, b: 3, c: 3, #0]`